projects/igniteui-angular/src/lib/directives/autocomplete/autocomplete.directive.spec.ts

```diff
@@ -369,7 +369,10 @@ describe('IgxAutocomplete', () => {
                     expect(dropdownListScrollElement.children[itemIndex].nativeElement.textContent.trim()).
                     toEqual(filteredTowns[itemIndex]);
                     const isFocused = itemIndex === 0 ? true : false;
-                    expect(dropdownListScrollElement.children[itemIndex].classes[CSS_CLASS_DROP_DOWN_ITEM_FOCUSED]).toEqual(isFocused);
+                    const hasFocusedClass =
```

Comment:
Can you change it to:
 const itemElement = dropdownListScrollElement.children[itemIndex].nativeElement;
                    expect(itemElement.textContent.trim()).
                    toEqual(filteredTowns[itemIndex]);
                    const isFocused = itemIndex === 0 ? true : false;
                    expect(itemElement.classList.contains(CSS_CLASS_DROP_DOWN_ITEM_FOCUSED)).toEqual(isFocused);
                    expect(dropDown.items[itemIndex].focused).toEqual(isFocused);